examples/project.yaml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-beta
+kind: Project
+metadata:
+  id: bitbucket-demo
+description: A project that demonstrates integration with Bitbucket
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/bitbucket
+    types:
+    - repo:forked
+    - pullrequest:created
+    - pullrequest:updated
+    labels:
+      repo: brigadecore/test-repo
+  workerTemplate:
+    defaultConfigFiles:
+      brigade.js: |-
+        const { events, Job } = require("@brigadecore/brigadier");
+
+        async function runPipeline(event) {
+          let fooJob = new Job("foo", "debian:latest", event);
+          fooJob.primaryContainer.command = ["echo"];
+          fooJob.primaryContainer.arguments = ["foo"];
+          await fooJob.run();
+
+          let barJob = new Job("bar", "debian:latest", event);
+          barJob.primaryContainer.command = ["echo"];
+          barJob.primaryContainer.arguments = ["bar"];
+          await barJob.run();
+        }
+
+        events.on("brigade.sh/github", "repo:forked", () => {
```

Comment:
Here and below we'd want `"brigade.sh/bitbucket"` as the source instead, right?

Reply:
Nice catch.